src/util/neptune.mjs

```diff
@@ -144,9 +147,6 @@ export class NeptuneLoader {
 		console.warn(`loading '${p_source}' from s3 bucket${p_graph? ` into ${p_graph}`: ''}...`);
 
 		// fetch job id
-		let si_job = a_body[0].payload.loadId;
-
-		// start polling job
-		return await this.check_job_status(si_job);
+		return a_body[0].payload.loadId;
```

Comment:
this changes the return type of this function which breaks functionality in other parts of this project. should be `return await wait_for_completion(a_body[0].payload.loadId)`